.storybook/main.js

```diff
@@ -5,6 +5,6 @@ module.exports = {
     '@storybook/addon-actions',
     '@storybook/addon-viewport',
     '@storybook/addon-storysource',
-    "storybook-addon-next-router",
```

Comment:
Rather than modifying an existing addon. It's better to add it to the next line:

```
...
"storybook-addon-next-router",
'@storybook/addon-mock'
```

Reply:
This PR should be moved to another PR, so this PR's goal is to only fix the padding.

Reply:
sure

Reply:
> Rather than modifying an existing addon. It's better to add it to the next line:
> 
> ```
> ...
> "storybook-addon-next-router",
> '@storybook/addon-mock'
> ```

it would be cuz of the git pull i think, I will fix this

Reply:
You can remove the discord bug solution entirely because it won't work, haha. I updated the issue with a working solution: https://github.com/garageScript/c0d3-app/issues/1758